packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testInspectDocument.st

```diff
@@ -0,0 +1,10 @@
+unit-tests
+testInspectDocument
+	
+	| possibleMorphs |
+	self editorModel inspectDocument.
+	possibleMorphs := Project current world submorphs.
+	possibleMorphs := possibleMorphs select: [: morph|  morph class = PluggableSystemWindow].
+	possibleMorphs := possibleMorphs select: [: morph| morph label = self document class name].
+	possibleMorphs := possibleMorphs select: [: morph|  morph model object = self editorModel targetDocument].
```

Comment:
Maybe something like the following would be more Smalltalk like but not sure
```suggestion
	possibleMorphs := possibleMorphs select: [:morph |  
	    (morph class = PluggableSystemWindow) 
	    and: [morph label = self document class name]
	    and: [morph model object = self editorModel targetDocumen]].
```

Reply:
Tried that but it didn't work, maybe something about evaluation oder of and: , as it will output MessageNotUnderstood label for ProgressMorph

Reply:
ok